docs/hosting/environment-variables/configuration-methods.md

```diff
@@ -165,7 +165,7 @@ export N8N_ENCRYPTION_KEY=<SOME RANDOM STRING>
 ### Execute all workflows in the same process
 
 !!! warning "Deprecated"
-	n8n deprecated `own` mode and the `EXECUTIONS_PROCESS` flag in version 0.227.0. They will be removed in a future release. Main mode is now the default, so this step is not needed for version 0.227.0 and above.
+	n8n deprecated `own` mode and the `EXECUTIONS_PROCESS` flag in version 0.227.0. They will be removed in a future release. Main mode is now the default, so this step isn't needed for version 0.227.0 and above.
```

Comment:
Are we sure this is correct? From a quick check I'm not seeing this deprecation among the [0.227.0 commits](https://github.com/n8n-io/n8n/compare/n8n@0.226.2...n8n@0.227.0).

Reply:
I'm fairly sure that's when we added the note we were deprecating it . . .  I'll rephrase to be safe